.vscode/settings.json

```
@@ -25,7 +25,7 @@
     }
   },
 
-  "[html][css][scss][json][yaml][sql][markdown]": {
+  "[html][css][scss][json][yaml][sql][markdown][jsonc][github-actions-workflow]": {
```

Nit, to keep similar languages adjacent to each other:

```suggestion
  "[html][css][scss][json][jsonc][yaml][github-actions-workflow][sql][markdown]": {
```